include/solvers/navier_stokes_scratch_data.h

```diff
@@ -112,6 +112,7 @@ class NavierStokesScratchData
     gather_void_fraction                     = false;
     gather_particles_information             = false;
     gather_temperature                       = false;
+    gather_ch                                = false;
```

Comment:
I'm not sure about the abbreviation `ch` that is used in several places. Don't you think it's better to be more explicit? Like writing always `cahn_hilliard` instead of `ch`? I know it's longer, but the code is growing a lot, and a t some point is hard to quickly know what ch means...

Reply:
I hoped it to be as recognizable as VOF but if you think it's better this way I'll change it